File: sample/sample.java
package main

import java.util.date;

@SampleAnnotation
public class Sample {

    private final static String MESSAGE = "hello, telin colorscheme!";

    public static void main(String[] args) {
        String msg = MESSAGE;
        // 挨拶の読み出し (comment sample)
        greeting(msg);

        int a = 12;
        boolean flag = true;

        // TODO サンプル
        Date d = new Date();

        if (d != null) {
            return;
        }
    }

    /**
     * JavaDocのサンプル
     * @param msg メッセージ
     */
    public static void greeting(String msg) {
        System.out.println(msg);
        return;
    }
}

File: sample/sample.js
'use strict';

const msg = "hello, telin colorscheme!";

let sample = new Sample(msg);
sample.greeting();

// Sample class and サンプルコメント
class Sample {
    constructor(msg) {
        this.msg = msg;
    }

    greeting() {
        console.log(this.msg);
    }
}

function add(a, b) {
    return a + b;
}

// アロー関数
var allowSample = (msg) => {
    console.log(msg);
}
